src/main/java/com/imagelab/DashboardController.java

```diff
@@ -0,0 +1,51 @@
+package com.imagelab;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.util.List;
+
+import javafx.fxml.FXML;
+import javafx.scene.input.*;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.shape.Circle;
+
+public class DashboardController {
+
+    private final DataFormat circleFormat = new DataFormat("javafx.scene.shape.Circle");
+
+    @FXML
+    private AnchorPane buidlanchorpane;
+
+    @FXML
+    private Circle circleoperation;
+
+    @FXML
+    private void handleDragDetection (MouseEvent event){
+        Dragboard db = circleoperation.startDragAndDrop(TransferMode.COPY);
```

Comment:
It is okay to call this "dragboard" as db could mean database as well. 
As a matter of fact, it is better and easier to keep the naming as much closer to the camel case version of the class name.